src/gn.cu

```diff
@@ -883,15 +898,10 @@ double network_monte_carlo(network_t n,
 
 // if CUDA, move the datastructures to Unified_Memory
 #ifdef USE_CUDA
-  network_t gpu_n = load_network_to_gpu(n);
-  experiment_set_t gpu_e = load_experiment_set_to_gpu(e);
+  n = load_network_to_gpu(n);
```

Comment:
I want to assign n and e with the data loaded from GPU because we need to use it again in the for loop in line 935